tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/util/SourceEditorUtils.java

```diff
@@ -475,6 +723,54 @@ private static ProcessorMetaData generateProcessorMetaData(Class<?> processorCla
                 processorMetaData.setParameters(parameterMetaDataList);
             }
 
+            List<SyntaxMetaData> syntaxList = new ArrayList<>();
+            String finalClipBoardSyntax = "";
+            String finalSyntax = "";
+            StringBuilder syntax = new StringBuilder();
+            StringBuilder clipBoardSyntax = new StringBuilder();
```

Comment:
Since you are returning the MetaDataObject, any reason to initiate the string builders here? Move them to private methods